docs/data_management/lts/tutorial/lts_tutorial.md

```diff
@@ -0,0 +1,119 @@
+---
+toc_depth: 3
+---
+# Basic Workflow with LST and s3cmd
+
+This tutorial covers setting up and using S3 with LTS, including installing s3cmd, configuring your environment, creating and managing S3 buckets, and setting access permissions on Cheaha.
+
+## Setting Up Your Environment
+
+### Install s3cmd within Anaconda Environment on Cheaha
+
+To interact with LTS using S3, you need the `s3cmd` tool installed. It is recommended to install it using `pip` within an Anaconda environment on Cheaha. Please avoid using `conda install s3cmd`, as that version may not function correctly. Instead, follow the below steps to install `s3cmd` using `pip`:
+
+First, access our interactive Open OnDemand (OOD) portal at https://rc.uab.edu and create an [interactive HPC Desktop session](../../../cheaha/open_ondemand/hpc_desktop.md). Once the HPC Desktop session lunched, open the terminal and run the below commands.
+
+```bash
```

Comment:
Maybe show screenshots of the expected output of the commands. We can discuss how to best present this. We can also cross-link to the Anaconda tutorial and documentation.

Reply:
Fixed.